configserver/src/main/java/com/yahoo/vespa/config/server/session/SessionPreparer.java

```diff
@@ -221,6 +223,16 @@ void checkTimeout(String step) {
             }
         }
 
+        void distributeApplicationPackage() {
+            FileRegistry fileRegistry = fileDistributionProvider.getFileRegistry();
+            FileReference fileReference = fileRegistry.addFile("");
```

Comment:
What does the empty string mean?

Reply:
It's the relative path in the application package, so it basically means add "add everything in the application package" directory. Should probably add a new method to make this clearer, I'll look into it in the next PR.